README.md

```diff
@@ -157,6 +157,9 @@ You'll need to upload these to an s3 bucket, then you can use these files to
 [create a simulation application](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-application.html), 
 and [create a simulation job](https://docs.aws.amazon.com/robomaker/latest/dg/create-simulation-job.html) in RoboMaker.
 
+## Generate Occupancy Map via map generation plugin
+Note: Procedural map generation is currently not supported in ROS2 (Dashing).
```

Comment:
Can we specify that it is because of `gazebo_ros_control` (https://github.com/ros-simulation/gazebo_ros_pkgs/wiki/ROS-2-integration#open_file_folder-gazebo_ros_control)?

Reply:
done